js/src/components/free-listings/configure-product-listings/__snapshots__/checkErrors.test.js.snap

```diff
@@ -1,5 +1,11 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`checkErrors Audience When the audience countries array is empty and the value of audience location option is 'selected', should not pass 1`] = `"Please select at least one country."`;
```

Comment:
💅  Out of this PR and opinionated

Maybe to consider migration to unit testing?

Reply:
Do you mean replacing `expect( errors.countries ).toBe( 'Please select at least one country.' );` with `expect( errors.countries ).toMatchSnapshot();` ? If so, some related discussions can be found in https://github.com/woocommerce/google-listings-and-ads/pull/893#discussion_r674040117 and https://github.com/woocommerce/google-listings-and-ads/pull/927#discussion_r681107540.